modules/modules.go

```diff
@@ -20,6 +20,9 @@ type module struct {
 
 	// is this module user visible (i.e. intended to be passed to `InitModuleServices`)
```

Comment:
This comment is now obsolete. (Modules passed to `InitModuleServices` are in fact "targets"). I suggest to remove second part of this comment.

Reply:
done in ea759669